demo/image_creation.sh

```diff
@@ -102,23 +102,21 @@ fi
 
 image=image-$epoch
 echo "*********************************************************************"
-echo "Deallocating and generalizing vm to capture image $image"
+echo "Capturing OsDisk snapshot of vm $vm with image $image"
 echo "*********************************************************************"
-az vm deallocate -g $rg -n $vm
-az vm generalize -g $rg -n $vm
-az image create -g $rg -n $image --source $vm --hyper-v-generation V2
-echo "Done"
+target_disk=$(az disk show --ids $(az vm show -g $rg -n $vm | jq .storageProfile.osDisk.managedDisk.id -r) | jq .name -r)
+az snapshot create -g $rg -n $vm-snapshot --source $target_disk
+az snapshot show -g $rg -n $vm-snapshot
```

Comment:
when the az snapshot create command succeeded, does it print out some information already? If it does, I don't think we need the az snapshot show

Reply:
Az snapshot create does -- think I was originally using it for debugging the snapshot_id but can easily get rid of the az snapshot show as it was just for debugging purposes as you noted.